com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -235,6 +235,8 @@ void BindMotionVectorPassColorBuffers(in RenderGraphBuilder builder, in PrepassO
                 // At this point in forward all objects have been rendered to the prepass (depth/normal/motion vectors) so we can resolve them
                 ResolvePrepassBuffers(renderGraph, hdCamera, ref result);
 
+                ApplyCameraMipBias(hdCamera);
```

Comment:
During the prepass we also sample normal map (this is use for SSR and SSAO). I "guess" we should then do it for depth prepass as well, what do you think?

Reply:
Good point. Yes lets add it to the depth prepass, since this is technically part of material sampling (due to normal sampling as you point out)

Reply:
Done!